src/menu.js

```diff
@@ -125,7 +126,12 @@ class Menu extends Component {
     const {inputValue} = this.autocomplete.state
     const {highlightedIndex} = this.state
     // eslint-disable-next-line no-unused-vars
```

Comment:
remove this and at the top of the `render` method put: `/* eslint no-unused-vars: 0 */` so we cover all of the items.